internal/lsp/completions/providers/regov1.go

```diff
@@ -0,0 +1,64 @@
+package providers
+
+import (
+	"strings"
+
+	"github.com/styrainc/regal/internal/lsp/cache"
+	"github.com/styrainc/regal/internal/lsp/types"
+)
+
+type RegoV1 struct{}
+
+func (*RegoV1) Run(c *cache.Cache, params types.CompletionParams) ([]types.CompletionItem, error) {
+	fileURI := params.TextDocument.URI
+
+	fileContents, ok := c.GetFileContents(fileURI)
+	if !ok {
+		// if the file contents is missing then we can't provide completions
+		return nil, nil
+	}
+
+	lines := strings.Split(fileContents, "\n")
+	if params.Position.Line >= uint(len(lines)) {
+		return nil, nil
+	}
+
+	line := lines[params.Position.Line]
+
+	if len(line) < int(params.Position.Character) {
+		return nil, nil
+	}
+
+	if !strings.HasPrefix(line, "import ") { // if in rule body
+		return nil, nil
+	}
+
+	words := strings.Split(line, " ")
+	lastWord := words[len(words)-1]
+
+	//nolint:gocritic
+	if !strings.HasPrefix("rego.v1", lastWord) {
```

Comment:
I'm not sure I understand this 😅 But it works so I must be missing something. But we exit if the last word doesn't start with "rego.v1"? 🤔

Reply:
I have added more notes for this in 37e711cbe1289749707b0c6fc75ef0c037bccdf3, does that make it more clear?

Reply:
Ah yes, it was the order of the arguments that I had missed 👍

Reply:
Yeah, it does seem like an odd order... but I think it's still valid! 😎

Reply:
Yes, clever :)